openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -0,0 +1,74 @@
+"""
+Django models related to sites.
+"""
+import collections
+
+from django.db import models
+from django.contrib.sites.models import Site
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from django_extensions.db.models import TimeStampedModel
+from jsonfield.fields import JSONField
+
+
+class SiteConfiguration(models.Model):
+    """
+    Model for storing site configuration related to comprehensive themes.
+
+    `site` field is one-to-one-key to django Site model
+    `values` field is json field to store theme configurations
```

Comment:
@mattdrayer  and I discussed this offline and I now understand that this field will contain what is currently defined in the MICROSITE_CONFIGURATION setting for a particular site.